example/src/pie/PieChartBasic.js

```diff
@@ -63,12 +64,33 @@ class PieChartBasic extends Component {
     }
 
     return (
-      <View>
-        <Pie
-          data={data}
+      <ScrollView horizontal={true} style={{flex:1,backgroundColor:'#F5FCFF'}} contentContainerStyle={{justifyContent:'center',alignItems:'center'}}>
```

Comment:
Not needed and was a bit accidental but I don't see it as harmful. It can actually be useful if someone changes the chart size beyond what can be rendered on one screen. Willing to change it back if you think it makes more sense?